docs/decisions/0003-learner-credit-management.rst

```diff
@@ -0,0 +1,41 @@
+3. Learner Credit Management
+============================
+
+Status
+******
+
+Accepted - July 2022
+
+Context
+*******
+
+* We decided to enable the consumption of Enterprise Offers in the Enterprise Learner Portal (TODO: link to LP ADR)
+* Enterprise admins need the ability to check the remaining balance and/or total balance redeemed
+  on their Enterprise offer, so they can plan their learning program accordingly.
+
+Decisions
+*********
+
+* Initially will only support "first-come, first-serve, org-wide budget" flavor of Offers.
```

Comment:
Might be worth calling out that the page is also available for enterprises with 1 offer.